app/scripts/background.js

```diff
@@ -1119,7 +1119,8 @@ export function setupController(
       senderUrl.pathname === phishingPageUrl.pathname
     ) {
       const portStreamForPhishingPage =
-        overrides?.getPortStream?.(remotePort) || new PortStream(remotePort);
+        overrides?.getPortStream?.(remotePort) ||
+        new PortStream(remotePort, { chunkSize: 0 });
```

Comment:
`chunkSize` disables chunking, effectively making `PortStream` identical to the `'extension-port-stream'` package.